### PR TITLE
update readme and changelog for aad scope updates

### PR DESCRIPTION
**Pull request by yiliuTo**
Update CHANGELOG.md and README.md about scope checking logic.

### CODE REVIEW

**Review by chenrujun — APPROVED** [review]
LGTM.

**Comment by backwind1233** [comment]
LGTM

**Comment by backwind1233** [comment]
LGTM.

File: sdk/spring/azure-spring-boot-starter-active-directory/README.md [hunk 1]
```diff
@@ -232,6 +232,13 @@ public class AADOAuth2LoginConditionalPolicyConfigSample extends WebSecurityConf
     }
 }
 ```
+### Customize scopes in authorize requests
+
+By default, `azure-spring-boot-starter-active-directory` configures scopes of `openid`, `profile` and `https://graph.microsoft.com/user.read` to implement OpenID Connect protocol and access of Microsoft Graph API. For customization of scope, developers need to configure in the `application.properties`:
+```yaml
```
Comment (saragluna) [inline sdk/spring/azure-spring-boot-starter-active-directory/README.md:238]: nit: this is not yaml format

Comment (yiliuTo) [reply sdk/spring/azure-spring-boot-starter-active-directory/README.md:238]: thanks for reminding, I will fix this in another pr

**Pull request merged**